src/core_plugins/metrics/public/components/vis_with_splits.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { getDisplayName } from './lib/get_display_name';
+import { last, findIndex, first } from 'lodash';
+import calculateLabel  from '../../common/calculate_label';
+export function visWithSplits(WrappedComponent) {
+  function SplitVisComponent(props) {
+    const { model, visData } = props;
+    if (visData[model.id].series.every(s => s.id.split(':').length === 1)) {
```

Comment:
@simianhacker I think that this change introduced a bug.
Before this PR, the `.series` property was being defaulted to `[]` [here](https://github.com/elastic/kibana/pull/13583/files#diff-9cdee9f60cfc8573eb26edc42c22b755R32), and now its not.

Maybe do something like this:
```js
import { get, last, findIndex, first } from 'lodash';

function SplitVisComponent(props) {
const { model, visData } = props;
const series = get(visData, `${model.id}.series`, []);
// ...
```

and erase all `visData[model.id].`